generator/internal/genclient/translator/protobuf/protobuf_test.go

```diff
@@ -485,32 +485,49 @@ func TestMapFields(t *testing.T) {
 
 func newCodeGeneratorRequest(t *testing.T, filename string) *pluginpb.CodeGeneratorRequest {
 	t.Helper()
-	contents, err := os.ReadFile(filepath.Join("testdata", filename))
+	tempFile, err := os.CreateTemp("", "protoc-out-")
 	if err != nil {
 		t.Fatal(err)
 	}
-	accessor := protocompile.SourceAccessorFromMap(map[string]string{
-		filename: string(contents),
-	})
-	compiler := protocompile.Compiler{
-		Resolver:       &protocompile.SourceResolver{Accessor: accessor},
-		MaxParallelism: 1,
-		SourceInfoMode: protocompile.SourceInfoStandard,
+	defer os.Remove(tempFile.Name())
+
+	var stderr bytes.Buffer
+	cmd := exec.Command("protoc",
```

Comment:
In the README for the generator can we provide a link to where to download this. I am fine doing this here but it then requires/assumes protoc in the host environment to run tests.

Reply:
Done.